File: sandbox.client/front/ts/model/Phone.ts
export class Phone {
  public id : number/*int*/;
  public clientId : number/*int*/;
  public number : string;
  public type : string;

  public isActive: boolean = true;

  constructor(clientId: number, number: string, type: string) {
    this.number = number;
    this.type = type;
  }
}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/Phone.java
package kz.greetgo.sandbox.controller.model;

public class Phone {
    public int clientID;
    public String number;
    public String phoneType;

    public ClientDetails toClientDetails(ClientDetails clientDetails) {
        if ("HOME".equals(this.phoneType)) {
            clientDetails.homePhone.add(this.number);
        } else
        if ("WORK".equals(this.phoneType)) {
            clientDetails.workPhone.add(this.number);
        } else {
            clientDetails.mobilePhones.add(this.number);
        }

        return clientDetails;
    }
}
